test/integration/cli-args/info/info.expected

```diff
@@ -28,22 +28,65 @@
           "dimensions": 0
         }
       ],
-      "dimensions": 1
+      "dimensions": 1,
+      "constraint": [
+        { "constraint": "none" },
+        { "constraint": "none" },
+        {
+          "constraint": [
+            { "constraint": "none" }, { "constraint": "none" }
+          ]
+        }
+      ]
     }
   },
   "parameters": {
-    "l": { "type": "real", "dimensions": 1 },
-    "m": { "type": "real", "dimensions": 1 },
-    "n": { "type": "real", "dimensions": 1 },
-    "o": { "type": "real", "dimensions": 1 },
-    "p": { "type": "real", "dimensions": 2 },
-    "q": { "type": "real", "dimensions": 2 },
-    "r": { "type": "real", "dimensions": 2 },
-    "s": { "type": "real", "dimensions": 2 },
-    "y": { "type": "real", "dimensions": 0 }
+    "low": {
+      "type": "real",
+      "dimensions": 0,
+      "constraint": { "lower": "0" }
+    },
+    "l": { "type": "real", "dimensions": 1, "constraint": "simplex" },
+    "m": { "type": "real", "dimensions": 1, "constraint": "unit_vector" },
+    "n": { "type": "real", "dimensions": 1, "constraint": "ordered" },
+    "o": {
+      "type": "real",
+      "dimensions": 1,
+      "constraint": "positive_ordered"
+    },
+    "p": { "type": "real", "dimensions": 2, "constraint": "covariance" },
```

Comment:
Kind of confused on the output format here. Is there a reason this is not `"type": "matrix"`?

Reply:
I'm frankly not sure, but that behavior has been the case since the flag was added in #810

My guess is that it let the output be non-recursive pre-tuples, because `array[N] matrix[M,K]` just got collapsed to `dimensions: 3`